### DISCUSSION TITLE
Using custom installer for talosctl upgrade

### DISCUSSION THREAD

**Discussion by jlowe64** [siderolabs/talos#7898] [Q&A] [answered]
Hello, 

I've built and uploaded my own installer adding zfs to my own ghcr.io repository and when I run the talosctl upgrade command I get the following error:

`talosctl upgrade --nodes $CONTROL_PLANE_01 --image ghcr.io/jlowe64/installer:v0.1.6`

The v0.1.6 is my own tag, but it uses v1.5.3 as a base when it's built. 

I get the following error:

`unknown command "ghcr.io/jlowe64/installer:v0.1.6" for "talosctl upgrade"`

When I add the `--debug` flag it doesn't seem to tell me any more information. Is there something I'm missing here? Should I be matching the version tag of the base installer?

My talosctl version is also v1.5.3.

**Comment by smira** [comment, marked as answer]
It feels like it for some reason doesn't treat that as a flag value, not sure if you have something else there, but this is something with your command line

**Reply by smira** [reply]
most probably your variable isn't set

**Reply by jlowe64** [reply]
@smira you were correct, there was an error in my .envrc. Sorry for the waste of time! Thank you for answering.
